src/routes/(app)/admin[[hash=admin_hash]]/+page.svelte

```diff
@@ -1,3 +1,116 @@
-<h1 class="text-3xl">Home</h1>
+<script lang="ts">
+	import { PUBLIC_VERSION } from '$env/static/public';
+	import { onMount } from 'svelte';
+	$: files = [];
+	$: lang = 'en';
 
-Soon, a lot of awesome things will be here.
+	async function fetchFiles() {
+		try {
+			const res = await fetch(`/docs/${lang}`);
+			files = await res.json();
+		} catch (error) {
+			console.error('Error while loading file :', error);
+		}
+	}
+	onMount(async () => {
+		fetchFiles();
+	});
```

Comment:
This should be in a `load` function, in `page.ts` (client-side): https://svelte.dev/docs/kit/load

To be able to refetch in a different language, the URL should be changed (either query param or url param) when changing language, and then the `load` function will automatically rerun